aspnetcore/signalr/hubs/sample/Hubs/ChatHub.cs

```diff
@@ -14,13 +14,13 @@ public class ChatHub : Hub
 
         public Task SendMessageToCaller(string message)
         {
-            return Clients.Caller.SendAsync("ReceiveMessage", message);
+            await Clients.Caller.SendAsync("ReceiveMessage", message);
```

Comment:
If we are going to take this, it won't compile due to the missing `async` on line 15 (same on line 20).